javagen/src/main/java/com/azure/autorest/model/clientmodel/PrimitiveType.java

```diff
@@ -266,4 +297,92 @@ public java.lang.String xmlSerializationMethodCall(java.lang.String xmlWriterNam
     public String toString() {
         return getName();
     }
+
+    private static class Builder {
```

Comment:
maybe we should move clientType, convertTo/FromClient to builder as well. though they already in the class and not hard to find, so maybe later.